maintainers/maintainer-list.nix

```diff
@@ -15401,4 +15401,10 @@
     github = "zuzuleinen";
     githubId = 944919;
   };
+  quasigod-io = {
```

Comment:
Please split the PR into two commits - one for adding the maintainer, one for the package.

Reply:
Corrected